CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Add `evpn_ethernet_segments` attribute to `iosxe_interface_ethernet` and `iosxe_interface_port_channel` resources and data sources
 - Add `set_ip_next_hop_unchanged` attribute to `iosxe_route_map` resource and data source
 - Enhance `set_communities` attribute documentation in `iosxe_route_map` to clarify support for well-known BGP community values (internet, local-AS, no-advertise, no-export, gshut)
+- Add support for maximum-path (ebgp and ibgp) for BGP ipv4 unicast with VRF
```

Comment:
Let's stick to the existing changelog format of "Add `xxx` attribute(s) to ...".